packages/yoastseo/index.js

```diff
@@ -18,6 +18,9 @@ import Paper from "./src/values/Paper";
 import AssessmentResult from "./src/values/AssessmentResult";
 import Assessment from "./src/scoring/assessments/assessment";
 import { DIFFICULTY } from "./src/languageProcessing/researches/getFleschReadingScore";
+import { Mark } from "./src/values";
```

Comment:
`Mark` is already exported above (through `import * as values from "./src/values";`), so you should be able to already fetch that in Premium 😄

Reply:
Thanks. I updated it.